src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -1085,13 +1085,8 @@ class FrontendGenImpl {
   }
 
   // construct JSON type from the argument type
-  // for example - a 3D array of f32 might produce something like
-  //     {"type" : "float" , "dims" : [4, 256, 16]}
-  // actually, this function would produce
-  //     {"type" : "f32" , "dims" : [4, 256, 16]}
-  //  for this example. The "f32" is mapped to "float" in getSignature, below
-  //
-  void concatTypeString(Type argType, llvm::raw_ostream &dstream) {
```

Comment:
I would not remove comments. If the comments are no longer valid due to code change, you should update the comments as well.

Reply:
@shizy818 Please keep the comment, updated for the current syntax.

Reply:
@AlexandreEichenberger @gongsu832 thanks, I'm going to update it.

Reply:
@gongsu832 Please check the comments update. thx.

Reply:
Comments look fine. Thanks.